core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/edit-ema-editor.component.html

```diff
@@ -71,12 +71,13 @@
                 data-testId="contentlet-tools" />
             <dot-ema-page-dropzone
                 *ngIf="
-                    es.editorData.canEditVariant &&
-                    !!es.bounds.length &&
-                    !es.editorData.device &&
-                    es.state === editorState.DRAGGING
+                    (es.editorData.canEditVariant &&
+                        !!es.bounds.length &&
+                        !es.editorData.device &&
+                        es.state === editorState.DRAGGING) ||
+                    es.state === editorState.SCROLLING
                 "
-                [containers]="es.bounds"
+                [containers]="es.state === editorState.SCROLLING ? [] : es.bounds"
```

Comment:
if you can go from `SCROLLING` to `DRAGGING` you don't need this logic

Reply:
U right, added that logic to updater!